source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,37 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-06-30
+
+### Highlights
+
```

Comment:
Collapsed Reply Threads (Early Beta Access)

We're excited to give you early access to Collapsed Reply Threads (Beta). It can be enabled in the System Console > Experimental > Collapsed Reply Threads (Beta)

Reply:
@esethna Based on [Confluence](https://mattermost.atlassian.net/wiki/spaces/Threads/pages/1972469761/Cloud+and+Self-Managed+Release+Plan), I thought we ship marketing for Cloud July 14th release?

Reply:
Ah, sorry to clarify: By Marketing I'm referring to blog posts, tweets, etc. I think it's fine to note in the changelog for cloud @amyblais